pages/ai-ecosystem/integrations.mdx

```diff
@@ -140,6 +141,26 @@ In the image below, you can see what's happening under the hood to get the answe
 
 ![llama-retriever](/pages/ai-ecosystem/integrations/llama-retriever.png)
 
+{<h3 className="custom-header">Demos</h3>} 
+
+If you'd like to take it one step further, explore how Memgraph and LlamaIndex
+work together in real-world applications with these interactive demos:
+
+- **Agentic GraphRAG system**: learn how to build an agent-powered graph retrieval-augmented generation (RAG) system using Memgraph and LlamaIndex.
+
+- **Multi-agent GraphRAG System**: dive into a more advanced setup with multiple agents collaborating in a GraphRAG system.
+
+<Cards>
+  <Cards.Card
+    title="Agentic RAG system"
+    href="https://github.com/memgraph/ai-demos/tree/main/integrations/llamaindex/agent%20rag%20system"
+  />
+  <Cards.Card
+    title="Multi-agent RAG system"
+    href="https://github.com/memgraph/ai-demos/tree/main/integrations/llamaindex/multi-agent%20rag%20system"
+  />
+</Cards>
```

Comment:
I wouldn't use cards, since you mentioned them above. Just add links to the above-mentioned (I know cards are nicer but to avoid confusion/duplication). Otherwise, looks great!